src/views/Player.vue

```diff
@@ -376,13 +393,20 @@ export default {
 
     async setPlayerProperties(player) {
       player.on("ready", () => {
+        this.player.currentTime = Math.abs(this.previousPlayerTime - 5)
+
         var progressBar = document.querySelectorAll(".plyr__progress")[0];
         this.plioQuestions.forEach(function (plioQuestion) {
           var question = plioQuestion.item;
           // Add marker to progress bar
           var marker = document.createElement("SPAN");
           marker.setAttribute("id", "marker")
           marker.classList.add("tooltip");
+
+          if (plioQuestion.state == "answered") {
+            marker.classList.add("tooltip-answered")
+          }
+          
```

Comment:
When initialzing the markers on the progress bar, check if a question has been answered before. If it is, then set the marker as green